skel/cmd/sonic/app.go

```diff
@@ -10,6 +11,7 @@ import (
 	"github.com/openware/sonic/skel/config"
 	"github.com/openware/sonic/skel/handlers"
 	"github.com/openware/sonic/skel/models"
+	"github.com/otiai10/copy"
```

Comment:
in this framework I'm against including too many dependencies

while copying we also need to replace the package name
from "github.com/openware/sonic/skel/models" to "NAME/models"

`import (
    "io/ioutil"
    "path/filepath"
    "os"
    "strings"
)

func copy(source, destination string) error {
    var err error = filepath.Walk(source, func(path string, info os.FileInfo, err error) error {
        var relPath string = strings.Replace(path, source, "", 1)
        if relPath == "" {
            return nil
        }
        if info.IsDir() {
            return os.Mkdir(filepath.Join(destination, relPath), 0755)
        } else {
            var data, err1 = ioutil.ReadFile(filepath.Join(source, relPath))
            if err1 != nil {
                return err1
            }
            return ioutil.WriteFile(filepath.Join(destination, relPath), data, 0777)
        }
    })
    return err
}`

Reply:
I have a question.

Will we copy all files and directories from 'skel' to the destination with a new NAME or just models in 'skel' ? 
I think we shouldn't copy all files. We should make some template directory for it.

Reply:
We have to move all the files inside `skel` folder and replace every import definition `openware/skel` with `NAME/skel`